### ISSUE TITLE
feat: Subtasks Templates

### ISSUE THREAD

**Issue by smoothwizz** [smoothwizz/doloper#97]
**Description**
## I would like to have the option of adding the same subtasks from a template to a task. 
```
1. Understand the Feature Task:

Review the feature task or user story to understand its requirements and objectives.
Discuss any ambiguities or questions with team members or stakeholders.


2. Branch Creation:

Create a new feature branch from the main development branch.
Ensure the branch name is descriptive and follows naming conventions.


3. Code Implementation:

Write code to implement the new feature or resolve the task.
Follow coding best practices and coding standards.
Maintain proper code organization and readability.
Write unit tests to validate the functionality (if applicable).
Ensure the code passes all existing tests.


4. Documentation:

Update or create documentation related to the feature or task.
Include information on how to use or configure the new functionality.
Ensure code comments are clear and concise.


5. Code Review:

Self-review the code to catch any issues or bugs.
Request a code review from one or more team members.
Address any feedback or suggestions received during the review.
Make necessary code changes and improvements based on the review.


6. Testing:

Test the feature or task thoroughly in a development environment.
Ensure it works as expected and doesn't introduce regressions.
Perform any necessary integration testing.


7. Build and Compile:

Ensure the code compiles without errors.
Verify that the build process runs smoothly.


8. Run Linters and Static Analysis:

Check the code against coding style guidelines and linters.
Address any linting errors or warnings.


9. Run Automated Tests:

Execute automated test suites to verify code functionality.
Ensure that all tests pass successfully.


10. Manual Testing:

- Perform manual testing to validate the feature or task.
- Confirm that user interactions and scenarios work correctly.


12. Review Checklist:

- Review the PR checklist for the project (if applicable).
- Ensure that all required tasks have been completed.


13. Commit and Push:

- Commit the final code changes with a meaningful commit message.
- Push the feature branch to the remote repository.


15. PR Creation:

- Create a Pull Request (PR) from the feature branch to the main development branch.
- Include a clear and descriptive title and description in the PR.


17. Continuous Integration (CI):

- Ensure that CI/CD pipelines are triggered for the PR.
- Confirm that automated tests run as part of the CI process.
- Address any build or test failures.


19. Additional Checks:

- Verify that the code coverage is sufficient.
- Check for security vulnerabilities (if applicable).
- Ensure any relevant documentation updates are included.


21. Review and Approval:

- Await code review and feedback from team members.
- Address any requested changes.
- Once approved, the PR is ready for merging.


22. Merge and Cleanup:

- Merge the PR into the main development branch.
- Delete the feature branch (if no longer needed).
- Update project tracking tools with the task status.


23. Post-Merge Actions:

- Verify that the feature/task is functioning as expected in the main branch.
- Close the associated feature/task in project management tools.
- Communicate the completion of the task to relevant stakeholders.


24. Deployment (if applicable):

- If the feature/task includes changes that need to be deployed, coordinate with the deployment team.


25. Documentation and Release Notes:
- Update release notes with information about the feature/task (if applicable).
- Ensure that user-facing documentation reflects the changes.
```
